File: src/main/java/com/github/bondarevv23/memorizer/model/Side.java
package com.github.bondarevv23.memorizer.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "side")
public class Side {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    @Column(name = "title")
    private String title;

    @Column(name = "text")
    private String text;

    @Column(name = "image_link")
    private String imageLink;
}

File: bot/src/main/kotlin/memorizer/model/redis/Side.kt
package memorizer.model.redis

import com.fasterxml.jackson.annotation.JsonProperty
import java.io.Serializable

data class Side (
    var title: String?,
    @JsonProperty("imageLink")
    var image: String?,
    var text: String?
) : Serializable {
    public constructor() : this(null, null, null)
}
